pkg/cloud/aws/actuators/machine/actuator.go

```diff
@@ -96,37 +96,51 @@ func machinesEqual(m1 *clusterv1.Machine, m2 *clusterv1.Machine) bool {
 	return m1.Name == m2.Name && m1.Namespace == m2.Namespace
 }
 
+// isNodeJoin determines if a machine, in scope, should join of the cluster.
 func (a *Actuator) isNodeJoin(scope *actuators.MachineScope, controlPlaneMachines []*clusterv1.Machine) (bool, error) {
```

Comment:
Maybe we should rename this method to something like `shouldJoin`?

Reply:
The convention I have seen for methods returning `bool` is starting with  `Has`, `Is`, `Can` or `Allow`. `shouldJoin` kinda falls out of the convention.